### PR TITLE
Recommendation

### PR DESCRIPTION
**Pull request by agoz17**
I added average calculations and getting current intent of the user to telegram.py. It is under recommendation branch because after this I am going to add similarity calculations.

### CODE REVIEW

File: bookbotproject/conversationtree/models.py [hunk 1]
```diff
@@ -56,12 +56,13 @@ class Book(models.Model):
     # Every book has unique isbn
     isbn = models.CharField(max_length=100, blank = True)
     # We may know the title of the book.
-    title = models.CharField(max_length=100)
+    title = models.CharField(unique=True, max_length=100)
```
Comment (theTytoAlba) [inline bookbotproject/conversationtree/models.py]: Titles can be the same but authors may be different, please do not make this field unique.

**Comment by agoz17** [comment]
@theTytoAlba , I removed your comment accidentally. But I found it from my mails it was like that:

"> @@ -56,12 +56,13 @@ class Book(models.Model):
     # Every book has unique isbn
     isbn = models.CharField(max_length=100, blank = True)
     # We may know the title of the book.
-    title = models.CharField(max_length=100)
+    title = models.CharField(unique=True, max_length=100

Titles can be the same but authors may be different, please do not make this field unique. "

My response was: 
I don't remember why I add this. I tested my branch without that condition we can remove it safely.

However if we have a case with same book title on different books, our recommendation code won't be able to give correct results in this case. Since we don't have real data and consequently real isbns I don't think testing these edge cases would add value to our project. If we don't try those test cases there won't be any problem. If anyone objects this, we need to change our rating and comment models.

**Comment by agoz17** [comment]
@theTytoAlba  I deleted the unique condition from Book model.

**Comment by theTytoAlba** [comment]
Looks nice, I am updating the title because this PR is not about recommendations yet. Merging.
